packages/did-document/src/lite/index.ts

```diff
@@ -71,6 +72,14 @@ class DIDDocumentLite implements IDIDDocumentLite {
   async ownerPubKey(did = this.did): Promise<string> {
     return this.resolver.readOwnerPubKey(did);
   }
+
+  async lastBlock(did: string): Promise<utils.BigNumber> {
+    return this.resolver.lastBlock(did);
+  }
+
+  async readFromBlock(did: string, from: utils.BigNumber): Promise<IDIDDocument> {
+    return this.resolver.readFromBlock(did, from);
```

Comment:
I hope this is reading from `changed(address)`.

Reply:
it reads from `from` up to `changed(address)`